src/renderer/components/ConfigureServer/ConfigureServer.scss

```
@@ -188,41 +191,30 @@
         margin-bottom: 0;
     }
 
     .ConfigureServer {
         .ConfigureServer__body .ConfigureServer__content {
             flex-direction: column;
+            max-width: 640px;
 
             .ConfigureServer__message,
             .ConfigureServer__card {
-                width: 640px;
             }
```

Stylelint (17.14.0)

[error] Empty block (block-no-empty)

(block-no-empty)